docs/reference/objects/product/index.md

```diff
@@ -55,13 +55,17 @@ string
 The dates of the product as a formatted string.
 e.g. `15 November 2024` for an experience with a duration of less than or equal to one day.
 e.g. `15 - 20 November 2024` for an experience with a duration of more than one day.
+If a product is a recurring experience, this returns the range based on the depature dates, and does not account for the experience duration.
+e.g. `November 2024` for an experience with multiple departure dates in the same month.
+e.g. `November 2024 - January 2025` for an experience with multiple departure dates over more than one month.
 
 ## `product.depart_on`
 {: .d-inline-block }
 timestamp
 {: .label .fs-1 }
 
 If the product has fixed dates this returns the start date of the event as a timestamp, this can then be used in conjunction with Liquid's [built-in filters](https://shopify.github.io/liquid/filters/date/).
+If a product is a recurring experience, this returns the start date of the next upcoming date if there is one, or the first date if all dates are in the past.
```

Comment:
Thinking about this, I wonder if the logic should be the following:

- First it attempts to return the next upcoming (or ongoing) date
- Failing that, if all dates are in the past, it returns the most recent date

Just thinking that it might be weird if the `depart_on` suddenly changes to a year ago once the most recent experience has just finished.